src/components/SelectV2/DesktopDropdown.tsx

```diff
@@ -0,0 +1,161 @@
+import styled from '@emotion/styled'
+import React, { SyntheticEvent, useRef, useState } from 'react'
+import { debounce } from '../../utils/debounce'
+import { OptionProps, SelectVariant } from './Select'
+import { Listbox } from './Listbox'
+
+const Wrapper = styled.div`
+  position: absolute;
+  inset: 0;
+  display: none;
+  user-select: none;
+
+  @media (hover: hover) {
+    display: block;
+  }
+`
+
+export type ValueProp = string | number | readonly string[]
+
+interface DesktopDropdownProps {
+  'aria-label': string
+  options: OptionProps[]
+  value?: ValueProp
+  onChange?: (event: SyntheticEvent, value?: ValueProp) => void
+  variant: SelectVariant
+}
+
+export const DesktopDropdown = ({
+  'aria-label': ariaLabel,
+  options,
+  value,
+  onChange,
+  variant,
+}: DesktopDropdownProps) => {
+  const listboxRef = useRef<HTMLDivElement>(null)
+  const [open, setOpen] = useState(false)
+  const [focussed, setFocussed] = useState(
```

Comment:
This indicates to me that it's a boolean, although it's an index right? Maybe `focussedElement` would be a better name, or anything other that indicates it's an index.

Reply:
We also have a `selected` prop and I think it makes more sense to have both `focussed` and `selected`. Talked to Glenn and decided to keep it as it is.